ci/sge/docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
     build:
       context: .
       dockerfile: Dockerfile-master
+      args:
+        - TRAVIS_PYTHON_VERSION
```

Comment:
So what will happen in local testing if TRAVIS_PYTHON_VERSION is not set?

Reply:
Current behavior uses 3.6.6, but I don't know that its explicitly supported conda behavior for `python=""  `